### DISCUSSION TITLE
How do I automate the action of pressing the "Enter", "Delete", & "Backspace" keys?

### DISCUSSION THREAD

**Discussion by ShlomoZee** [AutomaApp/automa#51] [Q&A]
I am slowly learning how to use Automa. However, I cannot figure out how I can use one of the steps to press the "enter", Delete, & "Backspace" keys.
How can I accomplish this?

Thanks!

**Comment by Kholid060** [comment]
On the trigger event block, select the `keydown` or `keyup` and in the options at the `key` and `code` input, write `Enter`, `Delete`, or `Backspace`
